user-management-and-permissions_5c07ccc604286304a71d0252.md

```diff
@@ -0,0 +1,35 @@
+Semaphore 2.0 is "mirroring" your GitHub organization and permissions access to the projects is in sync with your GitHub permissions.
```

Comment:
This is not entirely true atm. We do mimick permissions in terms of "does the user have access to this project". But we don't mimick permissions in terms of "what type of access does the user have on this project". Atm, all members have the same access to projects on Semaphore, regardless the level of access on Github. But, @radwo can double check this. :slightly_smiling_face:

Reply:
Milica is right :+1: if you can see (read access) the project on GH, you have access to it on Semaphore.